src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -67,6 +67,24 @@
       <Link>snappy64.dll</Link>
     </Content>
   </ItemGroup>
+
+  <PropertyGroup>
+    <ZstdBinaries>Core/Compression/Zstd/lib/win</ZstdBinaries>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(ZstdBinaries)/libzstd.dll">
+      <Pack>true</Pack>
+      <PackagePath>x64/native/windows</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(ZstdBinaries)/libzstd.dll">
```

Comment:
Why from here? This looks like some random person's repo.

Let's find the official source and take the library from the there.

Reply:
Fixed. I took the library(1.4.4) from the official release: https://github.com/facebook/zstd/releases

Reply:
@DmitryLukyanov It might be a good idea to manually verify that a test app targeting NET48, NET452, netcore3.1, and netcore2.1, consuming a locally created NuGet with these changes, is able to utilize Zstandard. 

I think we may also need to update the .targets file in a fashion similar to the way we did for Snappy (https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/MongoDB.Driver.Core.targets#L4)

Reply:
I've added it. Packaging works on:
`NET452`,
`NET472`,
`NETCORE2.2`
But it doesn't work on `NETCORE3.0`. I'm investigating it. The worst part that we have the same situation with snappy.. 
I don't have NET48, but taking the issue with `NETCORE3.0` into account. I will download it and check